src/current/_includes/releases/v24.2/v24.2.0.md

```diff
@@ -0,0 +1,144 @@
+- `storage.sstable.compression_algorithm_backup_storage`, `storage.sstable.compression_algorithm_backup_transport`: Added two new [cluster settings]({% link v24.2/cluster-settings.md %}), `storage.sstable.compression_algorithm_backup_storage` and `storage.sstable.compression_algorithm_backup_transport`, which in addition to the existing cluster setting `storage.sstable.compression_algorithm`, can be used to alter the compression algorithm used for various types of [SSTs]({% link v24.2/architecture/storage-layer.md %}#ssts). [#124245][#124245]
+
+<h5 id="v24-2-0-settings-removed">Settings removed</h5>
+
+- `kv.rangefeed.range_stuck_threshold`: Removed the stuck rangefeed cancel feature and its related [cluster setting]({% link v24.2/cluster-settings.md %}) `kv.rangefeed.range_stuck_threshold`, because it was only available in [non-mux rangefeeds]({% link v24.2/advanced-changefeed-configuration.md %}#mux-rangefeeds). Previously, the stuck rangefeed cancel feature was introduced to restart single rangefeeds automatically if they had not received KV updates for some time. [#125663][#125663]
+- `storage.value_blocks.enabled`: The `storage.value_blocks.enabled` [cluster setting]({% link v24.2/cluster-settings.md %}) has been removed; value blocks are always enabled. [#122164][#122164]
+
+<h5 id="v24-2-0-settings-with-changed-defaults">Settings with changed defaults</h5>
+
+- `kv.dist_sender.circuit_breakers.mode` has had its default changed to `liveness range only = 1`.
+- `sql.defaults.results_buffer.size` has had its default changed to `512 KiB`: The default value of the `sql.defaults.results_buffer.size` [cluster setting]({% link v24.2/cluster-settings.md %}) has been changed from 16KiB to 512KiB. This reduces the chance that clients using [`READ COMMITTED`]({% link v24.2/read-committed.md %}) transactions will encounter errors that cannot automatically be retried within CockroachDB. [#124633][#124633]
+- `sql.metrics.max_mem_stmt_fingerprints`, `sql.metrics.max_mem_txn_fingerprints` have had default changed to `7500`: The default values for the [cluster settings]({% link v24.2/cluster-settings.md %}) `sql.metrics.max_mem_stmt_fingerprints` and `sql.metrics.max_mem_txn_fingerprints` have been changed from `100000` to `5000`, thus lowering the default limits for in-memory statement and transaction fingerprints. [#123430][#123430]
```

Comment:
```suggestion
- `sql.metrics.max_mem_stmt_fingerprints` and `sql.metrics.max_mem_txn_fingerprints` have each had their defaults changed to `7500`: The default values for the [cluster settings]({% link v24.2/cluster-settings.md %}) `sql.metrics.max_mem_stmt_fingerprints` and `sql.metrics.max_mem_txn_fingerprints` have been changed from `100000` to `5000`, thus lowering the default limits for in-memory statement and transaction fingerprints. [#123430][#123430]
```
This could use a rewrite to say "changed from X to Y" but also I'm confused if the new value was changed to 7500 or to 5000.

Reply:
in v24.2.0 beta they're def at 7500 

```
select variable, value from [show cluster settings] where variable ilike '%sql.metrics.max_mem_%';
                     variable                    | value
-------------------------------------------------+---------
  sql.metrics.max_mem_reported_stmt_fingerprints | 100000
  sql.metrics.max_mem_reported_txn_fingerprints  | 100000
  sql.metrics.max_mem_stmt_fingerprints          | 7500
  sql.metrics.max_mem_txn_fingerprints           | 7500
(4 rows)
```

I bet the code was updated in the cockroachdb PR and the release note commit message was not 

I'll change everything to 7500 here 

thanks for calling this out!

Reply:
ah upon inspection it looks like maybe the bump to 7500 was in a later PR, either way it's adjusted here